### ISSUE TITLE
Upskill Task 28 - Accessibility

### ISSUE THREAD

**Issue by KalinYorgov** [KalinYorgov/react-native-upskill#28]
Accessibility refers to designing and developing apps that can be used by people with disabilities. This involves implementing features like proper labeling, color contrast, and assistive technology compatibility. The benefits of accessibility include a wider audience reach, improved user experience, and compliance with accessibility laws and guidelines.

With that in mind we should make improvements to our cart screens.

**Requirements:**

- Add headers in cart/empty cart screens with appropriate accessibility labels. They should be informative about the cart status - empty/how much products are added.
- After the screens are rendered for the first time focus automatically on these headers.
- Add accessibility labels and hints to the rest of your cart content.

**Hints:**

-  To test your app accessibility use TalkBack (android) or VoiceOver (iOS)

**Resources:**

- https://bitrise.io/blog/post/an-introduction-to-mobile-app-accessibility
- 
- https://reactnative.dev/docs/accessibility
